### ISSUE TITLE
[stable/wordpress] setting customHTAccessCM causes template render error

### ISSUE THREAD

**Issue by mrbrownt** [helm/charts#11713]
**Version of Helm and Kubernetes**:
Helm: 2.12.3
K8s: 1.11.7-gke.4

**Which chart**:
stable/wordpress

**What happened**:
Render error when setting customHTAccessCM

**How to reproduce it**:

```
helm install --name test --dry-run --debug --set customHTAccessCM="a-config-map" stable/wordpress
[debug] Created tunnel using local port: '54171'

[debug] SERVER: "127.0.0.1:54171"

[debug] Original chart version: ""
[debug] Fetched stable/wordpress to /Users/mrbrownt/.helm/cache/archive/wordpress-5.2.3.tgz

[debug] CHART PATH: /Users/mrbrownt/.helm/cache/archive/wordpress-5.2.3.tgz

Error: render error in "wordpress/templates/deployment.yaml": template: wordpress/templates/deployment.yaml:210:28: executing "wordpress/templates/deployment.yaml" at <{{template "wordpres...>: template "wordpress.customHTAccessCM" not defined
```

**Comment by juan131** [comment]
Hi @mrbrownt

I think the issue was already fixed on this other PR: #11697

Could you take a look?

**Comment by mrbrownt** [comment]
Looks like that has in fact fixed the issue
